src/index.js

```diff
@@ -1,12 +1,13 @@
+import './index.css';
+import navigation from './modules/navigation.js';
```

Comment:
- [ ] Your webpack app compiles with errors. This results in adding and viewing of the list of books not possible.
![image](https://user-images.githubusercontent.com/45002654/235477578-8765ed49-6871-4270-84d8-6b4d8f77f666.png)

The error above states the truth. In the file` navigation.js`, we have no function called `navigation`. Please address this.